tests/publiccloud/aistack_rbac_run.pm

```diff
@@ -0,0 +1,70 @@
+use Mojo::Base 'publiccloud::basetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::utils;
+use version_utils;
+
+sub run_python_script {
```

Comment:
this function is not used anywhere right now

Reply:
Yes, I will remove it, thanks.